app/main/routes.py

```diff
@@ -35,3 +35,8 @@ def user(username):
 
     user = Users.query.filter_by(username=username).first_or_404()
     return render_template('user.html', title='User Profile', user=user, form=form)
+
+
+@bp.route('/upload')
```

Comment:
I am guessing this is for testing purposes? 🤔

Reply:
Yes, this was indeed for testing purposes. This whole part should be removed with the Refactor PR.